websites/E/e-z/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "tsoxas",
+		"id": "787787652644601857"
+	},
+	"service": "E-Z",
```

Comment:
Rename the folder. Helpful [Stackoverflow question](https://stackoverflow.com/questions/6899582/i-change-the-capitalization-of-a-directory-and-git-doesnt-seem-to-pick-up-on-it).